go/vt/wrangler/vexec_test.go

```diff
@@ -151,15 +152,11 @@ func TestVExec(t *testing.T) {
 			if testCase.errorString == "" {
 				require.NoError(t, err)
 				for _, result := range results {
-					if !testCase.result.Equal(result) {
-						t.Errorf("mismatched result:\nwant: %v\ngot:  %v", testCase.result, result)
-					}
+					assert.True(t, testCase.result.Equal(result), "mismatched result:\nwant: %v\ngot:  %v", testCase.result, result)
```

Comment:
We should remove the `got:\n%+v\nwant:\n%+v` output, since `testify` prints the same kind of output anyway.